analysis/import/SourceCodeParser/script/sample-output.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+echo Can only be run after jar is created by "gradle build"
+echo ------------------------------------------------
+echo    FolderOutput
+echo ------------------------------------------------
+java -jar build/libs/codecharta-sourcecodeparser-1.13.0.jar src/test/resources/ -out=table
```

Comment:
The version string should change accordingly to the gradle.properties file which is the single point of truth for analysis versioning. Our release script updates only the gradle.properties version number.